internal/testcli/executor.go

```diff
@@ -273,13 +273,27 @@ func checkExit(t *testing.T, ch <-chan int, code int) {
 	}
 }
 
-// RunWithError runs command and checks that is exits with error.
+// RunWithError runs command and checks that is exits with error and exit code 1.
 func (e *Executor) RunWithError(t *testing.T, args ...string) {
 	ch := setExitFunc()
 	require.Error(t, e.run(args...))
 	checkExit(t, ch, 1)
 }
 
+// RunWithErrorContains runs command and checks that is exits with error and exit code 1.
+func (e *Executor) RunWithErrorContains(t *testing.T, msg string, args ...string) {
+	ch := setExitFunc()
+	require.ErrorContains(t, e.run(args...), msg)
+	checkExit(t, ch, 1)
+}
+
+// RunWithCheckError only for flag Action and Required check.
+// cli.ExitCoder doesn't implement wrapping properly, so we check for an error message.
+func (e *Executor) RunWithCheckError(t *testing.T, msg string, args ...string) {
```

Comment:
Documentation should be improved, check the grammar. It should be mentioned what's the difference between this method and `RunWithErrorContains`.